packages/blob/src/index.node.test.ts

```diff
@@ -541,14 +541,14 @@ describe('blob client', () => {
             callbackUrl: 'https://example.com',
           },
         },
-        // eslint-disable-next-line @typescript-eslint/require-await
         onBeforeGenerateToken: async (pathname) => {
+          await Promise.resolve();
```

Comment:
instead of awaiting a fake promise, we could remove the async and return a `Promise.resolve({
            metadata: pathname,
          })`

Reply:
We can, is there a difference with the fake promise? In both case we create a fake promise.

Reply:
it seems cleaner to return a promise

Reply:
oh but this is just the test 😅, fine with your solution then

Reply:
Yes and also I thought "Well await Promise.resolve() could be the authentication part" so perhaps even closer to real world with the fake promise here